### PR TITLE
Browser tests

### LINKED ISSUE #130: Browser Tests
The test suite needs to be run skipping any tests that uses chromedriver with `pipenv run pytest ./scraper/src -k "not _browser"`. Is this because the path is needed for the chrome driver? If that is the reason do you want them to be runnable? I could add an optional command line flag that lets you specify the path to the driver and make a fixture to set the environment variable which would make them runnable. Something like `pipenv run pytest ./scraper/src --chromedriver=/usr/local/bin/chromedriver`. The flag would be optional and wouldn't have to be set. This would allow browser tests to still be skipped as they are currently if someone doesn't have a chromedriver installed.

I'm also thinking the current browser tests aren't viable tests? I tried running them and most fail so I think if adding the flag for the driver path is something you want the tests would also need to be updated?

### PR DESCRIPTION
**Pull request by sanders41**
Closes #130

### CODE REVIEW

**Comment by bidoubiwa** [comment]
Hey @sanders41, I want to try out your PR more than just reviewing the code. Unfortunately, I don't have time this week. I will get back asap :)

**Comment by sanders41** [comment]
No worries, I figured you were busy 😄

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,14 +45,47 @@ pipenv run ./docs_scraper <path-to-your-config-file>
 pipenv install --dev
 # Linter
 pipenv run pylint scraper
-# Tests
-pipenv run pytest ./scraper/src -k "not _browser"
 ```
 
+If you have chromedriver install you can run the full test suite by passing the path to your
+chromedrive
```
Comment (bidoubiwa) [inline CONTRIBUTING.md:51]:
```suggestion
If you have [a `chromedriver`](https://sites.google.com/chromium.org/driver/), you can run the full test suite by passing the path to your `chromedriver`.
```

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥🔥

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**